### ISSUE TITLE
[Feature Request] Keybinds to change to specific Modes

### ISSUE THREAD

**Issue by Crystallyne** [FTBTeam/FTB-Ultimine#36]
I LOVE having the various options with Ultimine for Shapeless, Small Tunnel, and Small Square. But one thing I have found myself really wishing for is a way to more quickly change between the different modes.

I could see two possibilities for this, and would be happy with either one:

1) Having 2 keybinds, to cycle "up" and "down" in the list.

2) Having keybind options for each type of mode. Leave them blank by default, but have them available so players can set keybinds for their favorite modes, and then press a single key to activate that mode.

As it is currently, I want to be able to switch between modes faster/frequently, and I have to hold down two buttons AND scroll my mouse wheel to change modes. It just feels cumbersome when I'm wanting to change modes frequently.

**Comment by Possemaster** [comment]
Yes Please, as now this is very annoying actually to change modes.

**Comment by exetico** [comment]
+1 

Maybe even allow a "Mode1" + "Mode2" key, so it's possible to bind a more to them, at just use the normal controls to trigger that mode while mining.
